content/en/blog/_posts/2022-05-27-maxunavailable-for-statefulset.md

```diff
@@ -50,7 +50,7 @@ spec:
         app: nginx
     spec:
       containers:
-      - image: k8s.gcr.io/nginx-slim:0.8
+      - image: registry.k8s.io/nginx-slim:0.8
```

Comment:
```suggestion
      # image changed since publication (previously used registry "k8s.gcr.io")
      - image: registry.k8s.io/nginx-slim:0.8
```

@krmayankk FYI